src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -3125,6 +3129,7 @@ void WriteErrorHelper(Exception exception)
                             progress.PercentComplete = percentComplete;
                             progress.RecordType = ProgressRecordType.Processing;
                             WriteProgress(progress);
+                            }
```

Comment:
Indentation.

Reply:
@TheSpyGod it looks like indentation here and below lines 4003-4011 still needs to be addressed